### PR TITLE
feat: Add aliases for text color to allow for upstream flexibility

### PR DESCRIPTION
**Pull request by lukelooker**
Adds two new aliases (`body` & `title`) for `text5` and leverages those as default colors for a few key components:

- `body` - `Paragraph`, `UnorderedList`, `OrderedList`
- `title` - `Heading`

Also fixed issue where  `UnorderedList` & `OrderedList` accepted `color` property but didn't actually respond to it.

### Requirements

Please check the following items are addressed in your pull request (or are not applicable)

- [x] Includes test coverage for all changes
- [ ] Documentation updated
- [ ] i18n impacts
- [x] a11y impacts

### CODE REVIEW

File: packages/components/src/Text/Code.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ import { TextBase, TextBaseProps } from './TextBase'
 
 export const Code = styled(TextBase).attrs<TextBaseProps>(
   ({
-    color = 'text',
+    color = 'text5',
```
Comment (ElliotTheHuman) [inline packages/components/src/Text/Code.tsx:32]: Should this be `title` / `body`? Wasn't sure if the alias was supposed to be used here or not.

File: packages/components/src/UnorderedList/UnorderedList.tsx [hunk 2]
```diff
@@ -36,6 +36,7 @@ import {
   LayoutProps,
   position,
   layout,
+  textColor,
```
Comment (ElliotTheHuman) [inline packages/components/src/UnorderedList/UnorderedList.tsx:39]: Same issue as `OrderedList` where `UnorderedList` properly consumes the `color` prop, but gives a TS error saying it doesn't exist on the interface.

Comment (ElliotTheHuman) [reply packages/components/src/UnorderedList/UnorderedList.tsx:39]: Screen Shot 2021-02-03 at 5 08 05 PM

**Review by ElliotTheHuman — CHANGES_REQUESTED** [review]
I hit a couple interface / TS issues when testing `color` on `UnorderedList / OrderedList` but once that's resolved it should be good to go!

File: packages/components/src/OrderedList/OrderedList.tsx [hunk 3]
```diff
@@ -61,6 +36,7 @@ import {
   LayoutProps,
   position,
   layout,
+  textColor,
```
[the remark this answers is not in the document]
Comment (lukelooker) [reply packages/components/src/OrderedList/OrderedList.tsx]: Oddly - it actually works due to TS oddities but not within the Storybook stories (long story of house TS merges types on styled components). 

All of that said, explicitly adding `TextColorProps` is definitely the right solution - I went ahead and corrected it now.

File: packages/components/src/Text/Code.tsx (hunk 1 above)
Comment (lukelooker) [reply packages/components/src/Text/Code.tsx:32]: No - I think it's best to actually leave this one stuck to `text5`

File: packages/components/src/UnorderedList/UnorderedList.tsx (hunk 2 above)
Comment (lukelooker) [reply packages/components/src/UnorderedList/UnorderedList.tsx:40]: Same fix :)

File: packages/components/src/OrderedList/OrderedList.tsx [hunk 4]
```diff
@@ -22,31 +22,6 @@
...
- copies or substantial portions of the Software.
-
- THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
- SOFTWARE.
-
```
Comment (lukelooker) [inline packages/components/src/OrderedList/OrderedList.tsx:49]: Duplicate license entry

File: packages/design-tokens/src/system/index.ts [hunk 5]
```diff
@@ -55,7 +55,6 @@ export type {
   PaddingProps,
   PositionProps,
   SizeProps,
-  TextColorProps,
```
Comment (lukelooker) [inline packages/design-tokens/src/system/index.ts:58]: This is what was causing the type-mismatch error earlier.

**Review by ElliotTheHuman — APPROVED** [review]
LGTM!

**Pull request merged**